packages/animation/test.R

```diff
@@ -13,6 +13,13 @@ animation::saveVideo({
 	animation::brownian.motion(pch = 21, cex = 5, col = "red", bg = "yellow")
 }, video.name = "BM.mp4", other.opts = "-pix_fmt yuv420p -b 300k")
 
+animation::saveVideo({
+  par(mar = c(3, 3, 1, 0.5), mgp = c(2, 0.5, 0), tcl = -0.3, cex.axis = 0.8,
+      cex.lab = 0.8, cex.main = 1)
+  animation::ani.options(interval = 0.05, nmax = 300)
+  animation::brownian.motion(pch = 21, cex = 5, col = "red", bg = "yellow")
+}, video.name = "BM.mp4", other.opts = "-pix_fmt yuv420p -b 300k -c:v libx264")
```

Comment:
test x265 as well?

Reply:
Should be completed and pushed.  Testing locally now and Travis should check to confirm.